pandas/io/gbq.py

```diff
@@ -106,15 +112,17 @@ class TableCreationError(PandasError, ValueError):
     """
     pass
 
-
 class GbqConnector(object):
+    scope = 'https://www.googleapis.com/auth/bigquery'
 
-    def __init__(self, project_id, reauth=False):
+    def __init__(self, project_id, reauth=False, verbose=True, private_key=None):
```

Comment:
let's default to ``verbose=False``. you could have an option do this I suppose, e.g. ``pd.options.gbq.verbose``